cloudmailer/cloudmailer.py

```diff
@@ -569,92 +556,106 @@ def scheduleReboot(data,hostgroups, hostdict, starttime, interval):
-            emailcopy.write("Bcc: %s\n" % MAIL_BCC)
-        emailcopy.write("Subject: %s\n" % subject)
-        emailcopy.write(projmail_str)
-        emailcopy.close()
-
-        if send_emails and len(projects[project]["emails"]) > 0:
+        if send_emails:
             if ask_for_verification:
-                askToContinue('Are you sure that you want to send the emails?', 'Yes I am sure')
+                ask_to_continue('Are you sure that you want to send the emails?', 'Yes I am sure')
                 ask_for_verification = False
-            print ("Really sending emails to: %s" % ",".join(projects[project]["emails"]))
-            if MAIL_BCC:
-                print ("Really sending BCC emails to: %s" % MAIL_BCC)
-            for email_address in projects[project]["emails"]:
-                msg = MIMEText(projmail_str)
-                msg["Subject"] = subject
-                msg["To"] = email_address
-#                print(msg)
-                print ("Sending email to: %s" % email_address)
-                smtpconn.sendmail(MAIL_FROM, email_address, msg.as_string())
-                print ("Email to %s sent successfully" % email_address)
+
+            print(f"Really sending emails to: {emails_str}")
+            send_mails_to_list_of_emails(smtpconn, subject, project_email_address_list,
+                                         projmail_str)
+
             if MAIL_BCC:
-                for single_mail_bcc in MAIL_BCC.split(","):
-                    msg = MIMEText(message_bcc)
-                    msg["Subject"] = subject + " - " + projects[project]["name"]
-                    msg['To'] = single_mail_bcc
-#                    print(msg)
-                    print ("Sending BCC email to: %s" % single_mail_bcc)
-                    smtpconn.sendmail(MAIL_FROM, single_mail_bcc, msg.as_string())
-                    print ("BCC email to %s sent successfully" % single_mail_bcc)
-        elif notified_admin == False:
-            print("Attention!!! Not sending emails right now. Please, check the created files and when you are sure execute this same command with '--I-am-sure-that-I-want-to-send-emails' parameter.")
+                print (f"Really sending BCC emails to: { MAIL_BCC}")
+                bcc_subject =  subject + " - " + project_name
+
+                bcc_message = f"Mail sent to: {emails_str} \n------\n\n {projmail_str}"
+                send_mails_to_list_of_emails(smtpconn, bcc_subject, MAIL_BCC.split(","),
+                                             bcc_message)
+
+        elif notified_admin is False:
+            print("Attention!!! Not sending emails right now. Please, check the created files " \
+                  "and when you are sure execute this same command with " \
+                  "\"--I-am-sure-that-I-want-to-send-emails\" parameter.")
             notified_admin = True
     smtpconn.quit()
 
-def askToContinue(question, required_answer='Yes'):
+def ask_to_continue(question, required_answer='Yes'):
```

Comment:
the naming style of the function was consistently camel case. Any reason for switching to snake case only for the newly introduced functions?

Reply:
In the last PR I was asked to make it more compliant with pylint, this would be one step in that direction.

Reply:
Let's keep the code consistent in one file. I have nothing against either approach, but if we want to change to snake_case, let's do it for the whole file. Or then just have these in camel case